tests/provisioning-manifest-build.yml

```diff
@@ -1 +1,6 @@
 - karafCommand: "bundle:list"
+# uninstall only if it exists, otherwise it throws an error
+# need to hardcode version for uninstall
+- if: org.jahia.osgi.BundleUtils.getBundleBySymbolicName('server-availability-manager',null) != null
```

Comment:
What was the issue with requiring an install of a newer version on top of the one embedded inside Jahia?

Why wasn't it sufficient to just install it as part of the regular workflow.

Reply:
The issue with requiring an install of a newer version on top of the one embedded inside Jahia stems from the need to thoroughly test our probes. One of our tests involves ensuring that everything is functioning correctly with a severity level set to "low." In this context, the ModuleStateProbe plays a crucial role as it returns a status of "YELLOW" if at least one module is not started.

Reply:
You're right, I hadn't thought of this way of doing things, it's actually better to define this in env.run.sh. 
Fixed, thanks for the feedbacks :-)